pisa/core/bin_indexing.py

```diff
@@ -0,0 +1,243 @@
+"""
+Functions to retrieve the bin location of each elements
+of an array, inside a Container, based on its specified 
+output binning.
+
+Etienne Bourbeau
```

Comment:
move this to __author__

Also, please add standard license etc (check how other files look like)

Reply:
removed the name, but I don't see the standard license in the other core scripts (maps, container, base_stage...